apps/pie-docs/src/components/button/button.md

```diff
@@ -0,0 +1,56 @@
+---
+eleventyNavigation:
+    key: 'Button'
+    parent: Components
+    order: 1
+---
+
+## Overview
+The purpose of buttons is to provide a clear and intuitive way for users to interact with the interface and perform various actions. They enhance the user experience by making key functionalities easily accessible and identifiable, helping users navigate and interact with the application or website more efficiently.
+
+Buttons serve a wide range of purposes in user interfaces, such as submitting forms, confirming actions, navigating to different pages or sections, or executing specific functions.
+
+
+{% contentPageImage {
+    src:"../../assets/img/components/button/button-overview.svg",
+    alt: "A large primary button."
+} %}
+
+---
+
+## Resources
+
+{% resourceTable {
+    rows: [
+        {
+            resource: resourceTypes.COMPONENT,
+            link: "https://www.figma.com/file/R2rBfzJP0hG0MZorq6FLZ1/%5BCore%5D-Components-%E2%9A%AA%EF%B8%8F-%5BPIE-2.0%5D?type=design&node-id=727-0&mode=design&t=CQEMbUPtF2XxT6Nu-0",
+            status: statusTypes.AVAILABLE
+        },
+        {
+            resource: resourceTypes.DOCUMENTATION,
+            link: "https://www.figma.com/file/j1YKygEyhqZ6zKVxcHapn5/%5BCore%5D-Component-Documentation-%E2%9A%AA%EF%B8%8F-%5BPIE-2.0%5D?type=design&node-id=727-0&mode=design&t=P7rOGYbNedINS79O-0",
+            status: statusTypes.AVAILABLE
+        },
+        {
+            resource: resourceTypes.WEB_COMPONENTS,
+            status: statusTypes.PRE_RELEASE
+        },
+        {
+            resource: resourceTypes.VUE,
+            status: statusTypes.AVAILABLE
+        },
+        {
+            resource: resourceTypes.REACT,
```

Comment:
Should be able to add the links for react, web components and vue too

Reply:
oh whoops 😂 will do that now, thank you!